src/android/toga_android/widgets/detailedlist.py

```diff
@@ -42,14 +44,16 @@ def create(self):
             self.native.removeAllViews()
 
         scroll_view = android_widgets.ScrollView(self._native_activity)
+        self._scroll_view = android_widgets.ScrollView(
+            __jni__=java.NewGlobalRef(scroll_view))
```

Comment:
This seems like a pattern we're likely to need a lot (i.e., convert to global reference)... can/should we wrap this up as a utility method, either in Rubicon or in toga.android.libs?

Reply:
Yes - I think that this pattern is going to come up again, and that `rubicon.java` would be a great place to put it. Maybe `rubicon.java` can grow a function `def global_ref(o)` that makes a global ref out of `o`.

If you like that, one of us can file an issue on `rubicon.java. That bug can also say that once it's released, we should migrate Toga to use it.

Reply:
Opened beeware/rubicon-java#59 to track the feature request.